python/llm/example/CPU/LangChain/transformers_int4/README.md

```diff
@@ -0,0 +1,49 @@
+# Langchain examples
+
+The example here show how to use langchain RAG with `bigdl-llm`. This script rag2.py can analyze PDF file.
```

Comment:
The example is adapted from [langchain cookbook](https://github.com/langchain-ai/langchain/blob/master/cookbook/Semi_structured_multi_modal_RAG_LLaMA2.ipynb)